rd_workshop/styles/blocks/main.scss

```diff
@@ -0,0 +1,404 @@
+main {
+    .section-1 { 
+        margin: 0;
+        box-sizing: border-box;
+        background-color: $main-color;
+        height: 763px;
+        width: 1903px;
+
+        .grey-color {
+            height: 763px;
+            width: 912px;
+            background-color: $secondary-color;
+        }
+
+        .content {
+            box-sizing: border-box;
+            width: 1199px;
+            height: 509px;
+            border-radius: 40px;
+            border: 4px solid $secondary-color;
+            background: $main-color;
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            justify-content: center;
+            position: absolute;
+            left: 275px;
+            top: 940px;
+
+            .right-content {
+                width: 640px;
+                height: 509px;
+                margin-top: 90px;
```

Comment:
I have a little bit moved this block below each other

<img width="674" alt="image" src="https://github.com/alisa-bielenko/Robot_dreams/assets/13466381/346d3df4-987d-433f-9193-07dd917b6049">